README.md

```diff
@@ -137,12 +137,12 @@ For code contributions, review the following prerequisites:
 
 - [Docker](https://www.docker.com/)
 - [Operator Courier](https://github.com/operator-framework/operator-courier) is used to build, validate and push Operator Artifacts
-- [Operator SDK](https://github.com/operator-framework/operator-sdk) v1.11.0
-- [Go](https://golang.org/) v1.16 is installed.
-- [Golint dependency](https://pkg.go.dev/golang.org/x/lint/golint): go get -u golang.org/x/lint/golint
+- [Operator SDK](https://github.com/operator-framework/operator-sdk) v1.21.0
+- [Go](https://golang.org/) v1.17 is installed.
+- [Golint dependency](https://pkg.go.dev/golang.org/x/lint/golint): go install golang.org/x/lint/golint@latest
 - [Golangci-lint](https://golangci-lint.run/usage/install/)
 - [Python 3.x](https://www.python.org/downloads/) v3.x is installed
-- [Cekit](https://cekit.io/) 4.1.1+ is installed
+- [Cekit](https://cekit.io/) v4.0.0+ is installed
```

Comment:
```suggestion
- [Cekit](https://cekit.io/) v4.1.1+ is installed
```

Reply:
I kept what was on main branch.
Better to update it later, no?

Reply:
oki